nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Discovery.c

```diff
@@ -1073,7 +1073,9 @@ NFCSTATUS phNciNfc_UpdateDiscConfigParams(void *pNciHandle,
     uint8_t bIndex=1;
     PH_LOG_NCI_FUNC_ENTRY();
 
-    if(pPollConfig->PollNfcAActive)
+    if(pPollConfig->PollNfcAActive ||
+        ((pPollConfig->PollNfcFActive && pNciContext->ResetInfo.NciVer & PH_NCINFC_VERSION_MAJOR_MASK) ==
+         (PH_NCI2xNFC_VERSION & PH_NCINFC_VERSION_MAJOR_MASK)))
```

Comment:
In the NCI 2.0 Specification PollNfcAActive and PollNfcFActive are merged together under the NCI1x PollNfcAActive id.
In case the NFC Client tries to activate PollNfcFActive in NCI2.0, we are forcing PollNfcAActive Id.

I improved the code as well as added a comment in the next rc.